src/ray/object_manager/object_manager.h

```diff
@@ -67,7 +70,57 @@ class ObjectManagerInterface {
 };
 
 // TODO(hme): Add success/failure callbacks for push and pull.
-class ObjectManager : public ObjectManagerInterface {
+class ObjectManager : public ObjectManagerInterface,
+                      public rpc::ObjectManagerServiceHandler {
+  /// Implementation of object manager service
```

Comment:
```suggestion
  /// Implementation of `ObjectManagerServiceHandler`
```
and put this under `public:`?

Reply:
It's not only handler, and some other func for rpc service